vibi-dpu/src/github/webhook.rs

```diff
@@ -42,12 +42,11 @@ pub async fn add_webhook(repo_owner: &str, repo_name: &str, access_token: &str)
     if headers_map_opt.is_none() {
         return;
     }
-    let mut headers_map = headers_map_opt.expect("Empty headers_map_opt");
-    headers_map.insert("Accept", HeaderValue::from_static("application/vnd.github+json"));
```

Comment:
Why remove this? 
GitHub API docs mention having this header as a best practice.